### PR TITLE
feat: update extension options

### PR DESCRIPTION
**Pull request by frederickfogerty**
- added styling
- added message after successful save
- allowed empty API key to be saved so that API key can be deleted

FYI in the demo below, the validation logic was slightly changed so that "valid" was a valid key when saving, so I didn't have to expose an actual key when recording this video.

https://user-images.githubusercontent.com/615334/156173396-53883d1c-b254-42b1-b98e-cb12fa77183d.mov

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Current dependencies on/for this PR:
* next
  * **PR #317** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/317" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: src/extension/options.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+.button-action-large {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 10px 12px;
+
+  position: static;
+  /* width: 81px; */
```
Comment (luqven) [inline src/extension/options.css:65]: Nit, let's just delete this.

File: src/extension/options.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+#post-save-instructions {
+  margin: 16px 0px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 24px;
+  color: #475f72;
+  /* display: none; */
```
Comment (luqven) [inline src/extension/options.css:130]: Let's delete this.

**Review by luqven — APPROVED** [review]
LGTM

**Pull request merged**